src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -25,7 +25,9 @@ internal partial class ProjectState
         private readonly IReadOnlyList<DocumentId> _additionalDocumentIds;
         private readonly AsyncLazy<VersionStamp> _lazyLatestDocumentVersion;
         private readonly AsyncLazy<VersionStamp> _lazyLatestDocumentTopLevelChangeVersion;
-        private AnalyzerOptions _analyzerOptions;
+
+        // this will be initialized lazily.
```

Comment:
that I dont know, I just changed name. changing behavior should be consulted by the one who added it I believe. anyway, not dealing with it in this PR.